### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by petterill**
Formatting, issue tracking, documentation, table of contents, settings. 

Django settings is mostly copypasta from the excellent work by @vikoivun at https://github.com/City-of-Helsinki/linkedevents/blob/master/config_dev.toml.example with Respa-specific settings thrown in. I'm not 100% sure if all the settings are correct, so appreciated if someone can review.

### CODE REVIEW

**Comment by JsseL** [comment]
Yay! 👍 Good stuff.

File: README.md [hunk 1]
```diff
@@ -86,11 +108,36 @@ python manage.py resources_import --all kirjastot
 
 
 ### Settings
-- `RESPA_IMAGE_BASE_URL`: Base URL used when building image URLs in email notifications. Example value: `'https://api.hel.fi'`.
```
Comment (vikoivun) [inline README.md]: Add where the settings are set:
Settings are done either by setting environment variables named after the setting or adding them to `.env`-file in the project root. .env-file syntax is similar to TOML-files (INI-files), ie. KEY=VALUE pairs. Project root is the directory where this README is.

Comment (vikoivun) [reply README.md]: You can also set settings in local_settings.py, which allows you to set any variables whatsoever. However, some of the settings documented here are named differently in settings.py, especially authentication variables.

Comment (petterill) [reply README.md]: Very good, sir!

**Pull request merged**